dist/src/test/java/io/camunda/application/CamundaDockerIT.java

```diff
@@ -97,13 +68,65 @@ public void testStartCamundaDocker() throws Exception {
                         camundaContainer.getHost(),
                         camundaContainer.getMappedPort(MANAGEMENT_PORT),
                         "/actuator/health")))) {
-      assertThat(response.getCode()).isEqualTo(200);
-      assertThat(EntityUtils.toString(response.getEntity()))
+
+      // then
+      assertThat(healthCheckResponse.getCode()).isEqualTo(200);
+      assertThat(EntityUtils.toString(healthCheckResponse.getEntity()))
           .isEqualTo(
               "{\"status\":\"UP\",\"components\":{\"brokerReady\":{\"status\":\"UP\"},\"brokerStartup\":{\"status\":\"UP\"},\"brokerStatus\":{\"status\":\"UP\"},\"indicesCheck\":{\"status\":\"UP\"},\"livenessState\":{\"status\":\"UP\"},\"readinessState\":{\"status\":\"UP\"},\"searchEngineCheck\":{\"status\":\"UP\"}},\"groups\":[\"liveness\",\"readiness\",\"startup\",\"status\"]}");
     }
   }
 
+  private void startContainer(GenericContainer container) {
+    try {
+      container.start();
+    } catch (Exception e) {
+      fail(
+          String.format(
+              "Failed to start container.\n" + "Exception message: %s.\n" + "Container Logs:\n%s",
+              e.getMessage(), container.getLogs()));
+    }
+  }
+
+  private <T extends GenericContainer> T createContainer(Supplier<T> containerSupplier) {
+    T container = containerSupplier.get();
+    createdContainers.add(container);
+    return container;
+  }
+
+  private ElasticsearchContainer createElasticsearchContainer() {
+    return new ElasticsearchContainer(ELASTICSEARCH_DOCKER_IMAGE)
+        .withNetwork(Network.SHARED)
+        .withNetworkAliases(ELASTICSEARCH_NETWORK_ALIAS)
+        .withEnv("xpack.security.enabled", "false")
+        .withExposedPorts(ELASTICSEARCH_PORT);
+  }
+
+  private GenericContainer createCamundaContainer() {
+    return new GenericContainer<>(CAMUNDA_TEST_DOCKER_IMAGE)
+        .withExposedPorts(SERVER_PORT, MANAGEMENT_PORT, GATEWAY_GRPC_PORT)
+        .withNetwork(Network.SHARED)
+        .withNetworkAliases(CAMUNDA_NETWORK_ALIAS)
+        .waitingFor(
+            new HttpWaitStrategy()
```

Comment:
💡 Ah I have overseen this before. So you wait with this strategy until it is return 200 right?

Reply:
yes